src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderEvent.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class OffenderEvent {
```

Comment:
Eventually we should break this down into separate classes for each event type -- future work ...

Reply:
Agreed there is lots to improve!